### ISSUE TITLE
Queries on first loaded page are never fetched again

### ISSUE THREAD

**Issue by fdev** [apollographql/apollo-client-nextjs#56]
**Problem**
Using the SSR ApolloWrapper as defined in the [README](https://github.com/apollographql/apollo-client-nextjs/blob/main/package/README.md), any page that has `useSuspenseQuery` will be filled with data (fetched on the server) right away. That data, however, is never refetched no matter what `fetchPolicy` you define. The pages are not cached (`export const dynamic = 'force-dynamic'`), but the query never seems to be run again.

Queries on other pages are fetched as expected, it's just the first page that is loaded (regardless of which page that is) will keep displaying the results of its initial SSR fetch.

**How to reproduce**
I have set up a [CodeSandbox](https://codesandbox.io/p/sandbox/interesting-dew-v7ccyr) to demonstrate this specific issue.

1. Open page one (hard refresh).
2. Note the total votes.
3. In a separate tab, [increment the vote count](https://apollo-next-13-polls.vercel.app/cc).
4. Visit page two, and go back to page one.
5. We expect the total count to first show the cached value, and then be updated with the latest value (`cache-and-network`). 
6. The value is never updated. :x:

We can repeat step 3 and 4 indefinitely, but page one will always show the initial total votes from step 2.

When page one **is not** the initial page we load, we get the expected behavior on page one:

1. Open page two (hard refresh).
2. Visit page one.
3. Note the total votes.
4. In a separate tab, [increment the vote count](https://apollo-next-13-polls.vercel.app/cc).
5. Visit page two, and go back to page one.
6. We expect the total count to first show the cached value, and then be updated with the latest value (`cache-and-network`).
7. The value is updated. :heavy_check_mark:

**Video of issue occurring in CodeSandbox**

https://github.com/apollographql/apollo-client-nextjs/assets/1277208/2cf749ef-f419-4ea8-8cc2-3e77c0bb93e4

**CodeSandbox**
https://codesandbox.io/p/sandbox/interesting-dew-v7ccyr

**Comment by phryneas** [comment]
Thanks for the bug report! I believe I fixed it over in #59. Could you give that one a go and report back if it works for you?

```
npm i @apollo/experimental-nextjs-app-support@0.0.0-commit-473354a
```

**Comment by fdev** [comment]
> Thanks for the bug report! I believe I fixed it over in #59. Could you give that one a go and report back if it works for you?
> 
> ```
> npm i @apollo/experimental-nextjs-app-support@0.0.0-commit-473354a
> ```

Thanks for the quick reply and fix. After updating the package in the CodeSandbox it works as expected.
